applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_quasistatic_damped_U_Pw_scheme.hpp

```diff
@@ -36,9 +36,9 @@ class NewmarkQuasistaticDampedUPwScheme : public NewmarkQuasistaticUPwScheme<TSp
         : NewmarkQuasistaticUPwScheme<TSparseSpace,TDenseSpace>(beta, gamma, theta)
     {
         //Allocate auxiliary memory
-        int NumThreads = ParallelUtilities::GetNumThreads();
-        mDampingMatrix.resize(NumThreads);
-        mVelocityVector.resize(NumThreads);
+        int num_threads = ParallelUtilities::GetNumThreads();
```

Comment:
unsigned int or SizeType? a negative number of Threads would be strange.

Reply:
Agreed, but the function returns an int (and is located in core), so unless we want to convert it (implicitly or explicitly) to a SizeType, we can't change this.

Reply:
We could, however, commit to the type returned by `ParallelUtilities::GetNumThreads` (which is now an `int`, but it may be improved in the future). I would also make it `const`:
```suggestion
        const auto num_threads = ParallelUtilities::GetNumThreads();
```

Reply:
Good point, changed